modules/ROOT/pages/sap/sap-connector-xml-maven.adoc

```diff
@@ -0,0 +1,105 @@
+= SAP XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule application in XML either from the Studio XML editor 
+or from a text editor, perform the following actions to get access to the connector:
+
+. Add the namespace for the connector to your XML flow.
+. Add dependency information to your Maven `pom.xml` file.
+
+In the Studio visual editor, when you drag a connector from the palette to the 
+canvas, Studio automatically populates the XML code with the connector namespace and schema location.
```

Comment:
Do "connector namespace and schema location" equate to manually supplied namespace and dependency information?